### ISSUE TITLE
angular 8

### ISSUE THREAD

**Issue by DerekLiang** [ngrx-utils/ngrx-utils#75]
Is there any Angular 8 upgrade path?

Here is the error I am getting when running `ng update @angular/cli @angular/core @ngrx/effects@8.0.0-rc.0`.

```
Package "@ngrx-utils/store" has an incompatible peer dependency to "@ngrx/store" (requires "^6.0.0-beta.1", would install "8.0.0-rc.0").
Package "@ngrx-utils/store" has an incompatible peer dependency to "@angular/common" (requires "^6.0.0 || ^6.0.0-rc.0" (extended), would install "8.0.0").
Package "@ngrx-utils/store" has an incompatible peer dependency to "@angular/core" (requires "^6.0.0 || ^6.0.0-rc.0" (extended), would install "8.0.0").
```

**Comment by sandangel** [comment]
I will upgrade to angular 8 today. Thanks for reporting. It’s backward compatible so don’t worry

**Comment by sandangel** [comment]
hi @DerekLiang , I just published version 0.12.0 with angular 8 support. Please check it out and report if you have any issue

**Comment by ghost** [comment]
Seems like it worked, but got a weird problem from Angulars migration stuff that you might be able to assist me with:

```
    ** Executing migrations for package '@angular/core' **
            ------ Static Query Migration ------
            With Angular version 8, developers need to
            explicitly specify the timing of ViewChild and
            ContentChild queries. Read more about this here:
            https://v8.angular.io/guide/static-query-migration

            The template migration strategy uses the Angular compiler internally and therefore projects that no longer build successfully after the update cannot use the template migration strategy. Please ensure there are no AOT compilation errors.

            Could not setup migration strategy for "src/tsconfig.app.json". The following error has been reported:

            Error: error TS100: Unexpected value 'NgLetModule in /Users/<REDACTED PATH>/node_modules/@ngrx-utils/store/ngrx-utils-store.d.ts' imported by the module 'SharedModule in /Users/<REDACTED PATH>/shared.module.ts'. Please add a @NgModule annotation.


            Migration can be rerun with: "ng update @angular/core --from 7 --to 8 --migrate-only"

            ------------------------------------------------
```

The file `node_modules/@ngrx-utils/store/ngrx-utils-store.d.ts` looks like this:

```
/**
 * Generated bundle index. Do not edit.
 */
export * from './public-api';
```

Following it down into `node_modules/@ngrx-utils/store/lib/directives/ngLet.d.ts`:

```
import { TemplateRef, ViewContainerRef, OnInit } from '@angular/core';
export declare class NgLetContext {
    $implicit: any;
    ngLet: any;
}
export declare class NgLetDirective implements OnInit {
    private _vcr;
    private _templateRef;
    private _context;
    ngLet: any;
    constructor(_vcr: ViewContainerRef, _templateRef: TemplateRef<NgLetContext>);
    ngOnInit(): void;
}
export declare class NgLetModule {
}
```

Any pointers?

**Comment by sandangel** [comment]
Oh, I see. Let me publish a fix

**Comment by sandangel** [comment]
@monocl-oskar I published a fixed with 0.12.1

**Comment by ghost** [comment]
Thank you, that did the trick!
